pom.xml

```diff
@@ -74,7 +74,7 @@
 
     <profiles>
         <profile>
-            <id>npm-it</id>
+            <id>default</id>
```

Comment:
I think we should keep it for now and change in all components at the same time if that is the case. Also, I think default is not a good name for this profile.

Reply:
Changed back. I guess this should be reverted as well? https://github.com/vaadin/vaadin-board-flow/pull/110/files#diff-600376dffeb79835ede4a0b285078036L53-R53

Reply:
Yes, thanks.

Reply:
Changed back on Board as well.